ui/src/app/shared/services/projects-service.ts

```diff
@@ -117,7 +118,7 @@ export class ProjectsService {
         const update = paramsToProj(params);
         return requests
             .put(`/projects/${params.name}`)
-            .send({project: {...proj, spec: update.spec}})
+            .send({project: {...proj, spec: update.spec, metadata: {labels: update.metadata.labels}}})
```

Comment:
Somehow, updating the project with metadata will get an error "Unable to edit project: appprojects.argoproj.io "default" is invalid: metadata.resourceVersion: Invalid value: 0x0: must be specified for an update"

<img width="1362" alt="Screen Shot 2020-08-27 at 2 22 49 AM" src="https://user-images.githubusercontent.com/62673041/91422926-b27cdb00-e80c-11ea-8c21-a4d4402303ed.png">
.